### PR TITLE
Pull for fork - add PocketMine

### PR DESCRIPTION
**Pull request by magicalyak**
Adding a rock-on for using minecraft server on Rockstor.  the Minecraft server is the pocket edition which works on iphone, kindles and other mobile devices.  Using the rockon allows you to control the service through rockstor. I had to add the restart option because it always takes twice to get this up but it should be solid and should work great!

### CODE REVIEW

File: pocketmine.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "PocketMine": {
+        "containers": {
+            "pocketmine": {
+                "image": "magicalyak/pocketmine",
+                "launch_order": 1,
+                "opts": [
+		  [
+			"-it",
+			"--restart=always",
```
Comment (schakrava) [inline pocketmine.json]: Thank you @magicalyak . I have two questions here. 

1. Why ```-it```?

2. The rock-on framework automatically adds ```--restart=unless-stopped```, not sure if you've noticed it. Given that, any reason, why we'd want ```--restart=always```?

Comment (magicalyak) [inline pocketmine.json]: -it should be removed it's not needed and I need to set resets to 1 or 2 it always needs one restart. Will fix soon

Comment (magicalyak) [inline pocketmine.json]: Done

**Comment by magicalyak** [comment]
This definitely should be tested by someone else.  Pocketmine seems a little unstable at times (not this, but the program itself).  Plus it's a manual update to getting this to work.

**Comment by magicalyak** [comment]
> @magicalyak Hello yet again. I know this is way old but given the image may have improved in stability, assuming auto update, things may be better now.
> 
> Let me know what you want to do, otherwise I think this one is slated for closure soon.

This is now updated to the latest bedrock edition that just came out, they rewrote the server and it doesn't need restart anymore so that's removed.  Just need to test this once and should be good to merge (unless someone else wants to test).

**Comment by magicalyak** [comment]
Sounds good, I must have synced the whole dir.  let's do a new PR

**Pull request closed without merging**